packages/ui/src/components/GalleryMedia/GalleryMedia.tsx

```diff
@@ -45,24 +43,22 @@ export function GalleryMedia({ className, gallery, locale }: Props) {
             >
                 <p className="title-x-small text-gray-100">{name}</p>
                 <div className="flex items-center justify-between">
-                    {Boolean(images_number || videos_number) && (
+                    {Boolean(imagesNumber || videosNumber) && (
```

Comment:
I know this is a matter of preference, but can we please agree to write these conditions in a way how humans think and speak about numbers?

I never say `if (apples count) then`, I say `if (apples count greater than zero) then` :) 

```suggestion
                    {imagesNumber > 0 || videosNumber > 0 && (
```